viewer/packages/pointclouds/src/PointCloudNode.ts

```diff
@@ -13,7 +13,7 @@ import { createPointClassKey } from './createPointClassKey';
 import { PickPoint, PotreePointColorType, PotreePointShape, PotreePointSizeType } from './potree-three-loader';
 
 import { PointCloudAppearance } from './styling/PointCloudAppearance';
-import { RawStylableObject } from './styling/StylableObject';
+import { StylablePointCloudObjectCollection } from './styling/StyledPointCloudObjectCollection';
```

Comment:
I think this class is uncessary and that we should replace use with two arguments where necessary (i.e. two arguments for assigning styles, one for unassigning).

Reply:
I disagree - we have the same setup for CAD models, except we haven't bothered giving the type a name there. We need to store what object collection has received what style, and this seems to be the most convenient way of doing so.